jenkins-pipelines/longevity-10gb-3h-ipv6.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    params: params,
+
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-10gb-3h-ipv6.yaml',
```

Comment:
If we can enable ipv6 for testing only by assigning ip_ssh_connections to 'ipv6'.

Can we make it optional in original longevity-10gh-3h.jenkinsfile? then we don't need to duplicate all the jobs.

@roydahan what's your opinion?

Reply:
It would be nice to a have a jenkinsfile for each job we want run regularly. We didn't yet decided which. We added this option to all longevity pipelines (in this PR), we'll be able to select that in Jenkins when needed

Reply:
If we added that to the general pipeline, we don't need this one and its test yaml.

Reply:
@roydahan So - what is the decision?Remove this pipeline and add to the general?
I think we anyway will need to create separate Jenkins job for IPv6

Reply:
I'm ok to leave it as is.
It'll just verify the IPv6 configuration and general sanity.